gossip/src/cluster_info.rs

```diff
@@ -273,7 +273,7 @@ pub fn make_accounts_hashes_message(
 pub(crate) type Ping = ping_pong::Ping<[u8; GOSSIP_PING_TOKEN_SIZE]>;
 
 // TODO These messages should go through the gpu pipeline for spam filtering
-#[frozen_abi(digest = "FsZnSeTYNH7F51AxTaKUixXxjT6if2ThmPN1mhDWtXZM")]
+#[frozen_abi(digest = "Ctxue3UVFXXqnHoMVAPmfBoCy3Cyg7gNCYBY7Cg9P3so")]
```

Comment:
this is expected according to this: https://github.com/rust-lang/rust/pull/104265

the actual abi change is below:

```patch
diff --git a/tmp/abi8/cluster_info__Protocol_frozen_abi__test_abi_digest_FsZnSeTYNH7F51AxTaKUixXxjT6if2ThmPN1mhDWtXZM b/tmp/abi8/cluster_info__Protocol_frozen_abi__test_abi_digest_Ctxue3UVFXXqnHoMVAPmfBoCy3Cyg7gNCYBY7Cg9P3so
index a5d2d7bc63..59986592ba 100644
--- a/tmp/abi8/cluster_info__Protocol_frozen_abi__test_abi_digest_FsZnSeTYNH7F51AxTaKUixXxjT6if2ThmPN1mhDWtXZM
+++ b/tmp/abi8/cluster_info__Protocol_frozen_abi__test_abi_digest_Ctxue3UVFXXqnHoMVAPmfBoCy3Cyg7gNCYBY7Cg9P3so
@@ -232,25 +232,25 @@ enum Protocol (variants = 6)
                                                 primitive u8
                                             element u8
                                                 primitive u8
-                                field gossip: std::net::socket_addr::SocketAddr
+                                field gossip: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field tvu: std::net::socket_addr::SocketAddr
+                                field tvu: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field tvu_forwards: std::net::socket_addr::SocketAddr
+                                field tvu_forwards: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field repair: std::net::socket_addr::SocketAddr
+                                field repair: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field tpu: std::net::socket_addr::SocketAddr
+                                field tpu: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field tpu_forwards: std::net::socket_addr::SocketAddr
+                                field tpu_forwards: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field tpu_vote: std::net::socket_addr::SocketAddr
+                                field tpu_vote: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field rpc: std::net::socket_addr::SocketAddr
+                                field rpc: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field rpc_pubsub: std::net::socket_addr::SocketAddr
+                                field rpc_pubsub: core::net::socket_addr::SocketAddr
                                     primitive str
-                                field serve_repair: std::net::socket_addr::SocketAddr
+                                field serve_repair: core::net::socket_addr::SocketAddr
                                     primitive str
                                 field wallclock: u64
                                     primitive u64
@@ -1789,7 +1789,7 @@ enum Protocol (variants = 6)
                                             tuple (elements = 1)
                                                 element u8
                                                     primitive u8
-                                        element std::net::ip_addr::IpAddr
+                                        element core::net::ip_addr::IpAddr
                                             primitive str
                                 field sockets: __SerializeWith
                                     tuple (elements = 1)
@@ -2081,25 +2081,25 @@ enum Protocol (variants = 6)
                                                         primitive u8
                                                     element u8
                                                         primitive u8
-                                        field gossip: std::net::socket_addr::SocketAddr
+                                        field gossip: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tvu: std::net::socket_addr::SocketAddr
+                                        field tvu: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tvu_forwards: std::net::socket_addr::SocketAddr
+                                        field tvu_forwards: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field repair: std::net::socket_addr::SocketAddr
+                                        field repair: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tpu: std::net::socket_addr::SocketAddr
+                                        field tpu: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tpu_forwards: std::net::socket_addr::SocketAddr
+                                        field tpu_forwards: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tpu_vote: std::net::socket_addr::SocketAddr
+                                        field tpu_vote: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field rpc: std::net::socket_addr::SocketAddr
+                                        field rpc: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field rpc_pubsub: std::net::socket_addr::SocketAddr
+                                        field rpc_pubsub: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field serve_repair: std::net::socket_addr::SocketAddr
+                                        field serve_repair: core::net::socket_addr::SocketAddr
                                             primitive str
                                         field wallclock: u64
                                             primitive u64
@@ -3638,7 +3638,7 @@ enum Protocol (variants = 6)
                                                     tuple (elements = 1)
                                                         element u8
                                                             primitive u8
-                                                element std::net::ip_addr::IpAddr
+                                                element core::net::ip_addr::IpAddr
                                                     primitive str
                                         field sockets: __SerializeWith
                                             tuple (elements = 1)
@@ -3930,25 +3930,25 @@ enum Protocol (variants = 6)
                                                         primitive u8
                                                     element u8
                                                         primitive u8
-                                        field gossip: std::net::socket_addr::SocketAddr
+                                        field gossip: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tvu: std::net::socket_addr::SocketAddr
+                                        field tvu: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tvu_forwards: std::net::socket_addr::SocketAddr
+                                        field tvu_forwards: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field repair: std::net::socket_addr::SocketAddr
+                                        field repair: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tpu: std::net::socket_addr::SocketAddr
+                                        field tpu: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tpu_forwards: std::net::socket_addr::SocketAddr
+                                        field tpu_forwards: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field tpu_vote: std::net::socket_addr::SocketAddr
+                                        field tpu_vote: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field rpc: std::net::socket_addr::SocketAddr
+                                        field rpc: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field rpc_pubsub: std::net::socket_addr::SocketAddr
+                                        field rpc_pubsub: core::net::socket_addr::SocketAddr
                                             primitive str
-                                        field serve_repair: std::net::socket_addr::SocketAddr
+                                        field serve_repair: core::net::socket_addr::SocketAddr
                                             primitive str
                                         field wallclock: u64
                                             primitive u64
@@ -5487,7 +5487,7 @@ enum Protocol (variants = 6)
                                                     tuple (elements = 1)
                                                         element u8
                                                             primitive u8
-                                                element std::net::ip_addr::IpAddr
+                                                element core::net::ip_addr::IpAddr
                                                     primitive str
                                         field sockets: __SerializeWith
                                             tuple (elements = 1)
```